### PR TITLE
Introduce a simple sample policy

### PR DESCRIPTION
**Pull request by dburgener**
This policy provides a simple system policy which is bootable in enforcing mode on Fedora 36 and is useful for demonstrations and testing.

### CODE REVIEW

File: data/policies/full_system/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+This directory contains a sample full system cascade policy intended for
+demonstrating and testing Cascade.
+
+The intent is to create a policy that boots in enforcing mode and allows simple
```
Comment (matt-sheets) [inline data/policies/full_system/README.md:4]: We should also say which distro & version this was tested against.  Maybe any big configuration changes that were made.

File: data/policies/full_system/all_files.cas [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	fs_context(this, "devtmpfs", trans);
+	fs_context(this, "tmpfs", trans);
+	fs_context(this, "hugetlbfs", trans);
+	fs_context(this, "devpts", trans);
+	fs_context(this, "mqueue", trans);
+	fs_context(this, "pipefs", task);
+	fs_context(this, "sockfs", task);
+}
+
+// TODO: below derive may not be working?
```
Comment (matt-sheets) [inline data/policies/full_system/all_files.cas:37]: Do we have an answer for this TODO yet?

Comment (dburgener) [reply data/policies/full_system/all_files.cas]: I had worked around it, and thought I removed it, but I guess not.  I just took it out.  The whole built in initial sid thing will be going away eventually.

File: data/policies/full_system/README.md (hunk 1 above)
Comment (dburgener) [reply data/policies/full_system/README.md:6]: Added a line, and also fleshed out the README.md quite a bit.

**Review by matt-sheets — APPROVED** [review]

**Pull request merged**